src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        final InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
Well we already do the check on the server (still not sure if its working), The only added network traffic would be sending the chat message. Technically we would be saving client side processing (a trivial amount)